src/Saunter/Serialization/IAsyncApiDocumentSerializer.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using Saunter.AsyncApiSchema.v2;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Saunter.Serialization
 {
     public interface IAsyncApiDocumentSerializer
     {
         string ContentType { get; }
 
-        string Serialize(AsyncApiDocument document, AsyncApiOptions options);
+        string Serialize(AsyncApiDocument document);
```

Comment:
I think AsyncApiOptions should contain the spec generation options which should not be used/needed for serialization.